docs/documentation/stories/include-angular-material.md

```diff
@@ -0,0 +1,152 @@
+# Include [Angular Material](https://material.angular.io)
+
+[Angular Material](https://material.angular.io) is a set of Material Design components for Angular apps. 
+This guide will walk you through adding material design to your Angular CLI project and configuring it to use Angular Material.
+
+Create a new project and navigate into the project...
+```
+ng new my-app && cd my-app
+```
+
+With the new project created and ready, you will next need to install Angular Material.
+
+Install the `@angular/material` library and add the dependency to package.json...
+```bash
+npm install --save @angular/material
+
+```
+
+Import the Angular Material NgModule into your app module...
+```javascript
+//in src/app/app.module.ts -->
+
+import { MaterialModule } from '@angular/material';
+// other imports 
+
+@NgModule({
+  imports: [
+    ...
+    MaterialModule.forRoot()
+  ],
+  ...
+})
+
+```
+
+Now that the project is set up, it must be configured to include the CSS for a theme. Angular Material ships with some prebuilt theming, which is located in `node_modules/@angular/material/core/theming/prebuilt`.
```

Comment:
`prebuilt/` is a directory, you should end that path with an extra `/` to indicate so.